pygmt/helpers/decorators.py

```diff
@@ -149,6 +149,13 @@
                   column value must exceed *gap* for a break to be imposed.
                 - **+p** - specify that the current value minus the previous
                   value must exceed *gap* for a break to be imposed.""",
+    "grid": r"""
+        grid : str or xarray.DataArray
+            Name of the input grid file or the grid loaded as a
+            :class:`xarray.DataArray` object.
+
+            For name of the input grid file, see
+            :gmt-docs:`gmt.html#grd-inout-full` for available modifiers.""",
```

Comment:
Explaining all available modifiers in the docstrings makes the docstrings too long. Instead, I added a link to the full description here.